docs/slides/content/tech-exercise-ii.md

```diff
@@ -247,27 +246,29 @@ Choose your own adventure but both half of the teams must be successful to finis
 
 
 #### Feedback
-* Q&A
+* Q & A
 * Real World Stories
 * Pull Requests Welcome 🦄
 
 
 
 #### WHO - Pipelines <!-- .element: class="title-bottom-left" -->
 <!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->
+<!--Tech Edit - this slide is empty except for the heading. -->
```

Comment:
@springdo do you have any screenshots you can upload? I also can dig a bit deeper and bring out something :D